### PR TITLE
Fix issue REN-25

### PR DESCRIPTION
**Pull request by aardschok**
* Visual patch to improve labels of buttons.
* Added start for save and load queue ( reuse looks from scene A which are also in scene B and C )

### CODE REVIEW

**Comment by BigRoy** [comment]
The save logic doesn't do anything - it's not triggering. Plus it can't currently be loaded. So I'm assuming this PR isn't ready to merge?

File: mayalookassigner/app.py [hunk 1]
```diff
@@ -109,8 +113,8 @@ def setup_ui(self):
 
         # Method buttons
         method_buttons_layout = QtWidgets.QHBoxLayout()
-        assign_to_selected_btn = QtWidgets.QPushButton("Assign to Selected")
-        assign_to_all_btn = QtWidgets.QPushButton("Assign to All")
+        assign_to_selected_btn = QtWidgets.QPushButton("Process Selected Queue")
```
Comment (BigRoy) [inline mayalookassigner/app.py]: Actually, this label might be more confusing? It's not actually doing anything with the queue with this button? Correct?

It's only assigning the things of the top view (that is not the queue) to what is currently selected. Or am I misreading the code?

**Comment by aardschok** [comment]
Updated PR:
* New buttons for load and save of queue
* Enabled save and load logic

File: mayalookassigner/models.py [hunk 2]
```diff
@@ -20,6 +20,7 @@ def add_items(self, items):
         self.beginResetModel()
 
         for item in items:
+            print item["asset"]
```
Comment (BigRoy) [inline mayalookassigner/models.py]: Print all the things! :) Hehe. Probably forgot this one.

**Comment by BigRoy** [comment]
Looking good.

**Pull request merged**